Development/ssl/test/ssl_test.cpp

```diff
@@ -0,0 +1,39 @@
+// The first "test" is of course whether the header compiles standalone
+#include "ssl/ssl_utils.h"
+
+#include "bst/test/test.h"
+#include "sdp/json.h"
+
+////////////////////////////////////////////////////////////////////////////////////////////
+BST_TEST_CASE(testSslParseErrors)
+{
+    ssl::experimental::X509_NAME_ptr x509_name(X509_NAME_new(), &X509_NAME_free);
+
+    struct test
+    {
+        std::string field;
+        std::string value;
+    };
+
+    const test tests[] =
+    {
+        { "C", "UK"},
+        { "O", "Disorganized Organization"},
+        { "CN", "Joe Bloggs"}
```

Comment:
In addition to these, I think you need some test values that contain characters that need escaping, at least comma, equal, plus, new line and backslash.

```suggestion
        { "C", "UK" },
        { "O", "Disorganized Organization" },
        { "CN", "Joe Bloggs" }
```